src/main/java/dao3/model/Manufacturer.java

```diff
@@ -0,0 +1,43 @@
+package dao3.model;
+
+public class Manufacturer {
+    private long id;
+    private String name;
+    private String country;
+
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    @Override
+    public String toString() {
+        return "Manufacturer{"
+                + "id=" + id + ", name='" + name + '\''
+                + ", country='" + country + '\'' + '}';
+    }
+}
```

Comment:
```suggestion
}
```
and you should add equals and hashCode methods